src/main/java/org/apache/maven/plugins/toolchain/ToolchainMojo.java

```diff
@@ -62,9 +62,9 @@ public class ToolchainMojo extends AbstractMojo {
     /**
      * Toolchains requirements, specified by one
      * <pre>  &lt;toolchain-type&gt;
-     *    &lt;param&gt;expected value&lt;/param&gt;
-     *    ...
-     *  &lt;/toolchain-type&gt;</pre>
+     *     &lt;param&gt;expected value&lt;/param&gt;
+     *     ...
+     *   &lt;/toolchain-type&gt;</pre>
```

Comment:
what about something such 
```
     * <pre>{@code  <toolchain-type>
     *     <param>expected value</param>
     *     ...
     *   </toolchain-type>
     * }</pre>
     *
```
I find more human readable compared to the escaped values. Should work for the goal page generation.

Reply:
if it works, I take: I'll try and report

Reply:
it works, good idea integrated, thank you :)